apiary.apib

```diff
@@ -216,9 +297,9 @@ or set account properties or destroy their accounts.
 
 Destroying an account, and changing account properties
 like username and password, require to pass the user's
-password as `x-password` header.
+bearer token as `Authorization` header.
```

Comment:
I'll remove this paragraph. The idea was some time back that destroying an account and changing account information would require extra authentication, but we at some point decided to remove this entirely and make it app logic. E.g. we could introduce [sudo mode](https://github.com/blog/1513-introducing-github-sudo-mode) or do something similar for Hoodie, or make it configurable, I dunno, shouldn't block us now